### DISCUSSION TITLE
Cadwork freezed and crashed when elements in console are selected.

### DISCUSSION THREAD

**Discussion by faure-levoux** [cwapi3d/cwapi3dpython#169] [General]
Hi, 
When I want to select some text in Cadwork's console to copy, the software is freezing and crashing.
Do you know why ? 

Thanks !

**Comment by Brunner246** [comment]
1. Mark text in the console to stop the process (cadwork ist blocked)
2. Copy the text
3. "free" the console with key `ESC` 

Closing the console, kills the 3d process.

**Comment by swatkinson** [comment]
Hi, not sure if you've fixed this issue, but if you or anyone else is having issues with this

Control + C ends processes in the terminal, so if you are highlighting text and hitting ctrl + C, you are closing the terminal which causes cadwork to crash.

To copy, you can just highlight the text, and then right click anywhere on the terminal, and the text will be copied to the clipboard :)
